src/tests/unit/tests/DetailsView/components/save-assessment-button.test.tsx

```diff
@@ -36,71 +44,95 @@ describe('SaveAssessmentButton', () => {
             href: 'url',
             userConfigurationStoreData,
         };
-        eventStub = new EventStubFactory().createMouseClickEvent();
     });
 
     describe('on dialog enabled', () => {
-        let wrapper: ShallowWrapper;
+        describe('render', () => {
+            beforeEach(() => {
+                wrapper = render(<SaveAssessmentButton {...propsStub} />);
+                fireEvent.click(wrapper.getByRole('link'));
+            });
+            it('snapshot of dialog', () => {
+                wrapper = render(<SaveAssessmentButton {...propsStub} />);
```

Comment:
We don't need to `render` here because it's already rendered in the `beforeEach`
```suggestion
```